modules/test/playwright/tests/frontend-data-set-web/main/pages/FDSSamplePage.ts

```diff
@@ -138,7 +138,13 @@ export class FDSSamplePage {
 		this.visualizationModeSelector = page.getByLabel('Show View Options');
 	}
 
-	async changeVisualizationMode(visualizationMode: VisualizationMode) {
+	async changeVisualizationMode({
```

Comment:
Unifying our two utils `changeVisualizationMode` and `selectVisualizationMode`, that were doing the same thing.